codeit/classes/Article.js

```diff
@@ -0,0 +1,13 @@
+class Article {
```

Comment:
Article은 export가 없어서 다른 파일들에서 import로 사용할 수 없을 것 같아요.
```suggestion
export class Article {
```